<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog_Profile</class>
 <widget class="QDialog" name="Dialog_Profile">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>718</width>
    <height>344</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#Dialog_Profile {
  font-family: &quot;Courier New&quot;;
  background-color: #FFFFFF;
}
QGroupBox {
  font-family: &quot;Courier New&quot;;
  font-size: 13px;
  background-color: rgba(248, 249, 250, 1);
}
QLineEdit {
  padding: 4px 0;
  border: 1px solid #ced4da;
  border-radius: 8px;
  margin-left: 8px;
  padding-left: 4px;
  background-color: #FFFFFF;
  color:#212529;
}
QPlainTextEdit {
	background-color: #FFFFFF;
  color:#212529;
}
QLabel {
  font-family: &quot;Courier New&quot;;
  font-size: 13px;
  color: rgb(90, 93, 97);
}
QRadioButton {
  font-family: &quot;Courier New&quot;;
  font-size: 13px;
  color: #212529;
}
QComboBox {
  font-family: &quot;Courier New&quot;;
  font-size: 13px;
  color: #212529;
}
QPushButton {
  color: #212529;
}

QCheckBox{
  color: #212529;
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="section_1_layout">
     <item>
      <widget class="QGroupBox" name="uid_container">
       <property name="title">
        <string/>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <property name="leftMargin">
         <number>4</number>
        </property>
        <property name="topMargin">
         <number>8</number>
        </property>
        <property name="rightMargin">
         <number>8</number>
        </property>
        <property name="bottomMargin">
         <number>4</number>
        </property>
        <item>
         <layout class="QVBoxLayout" name="uid_container_layout">
          <property name="spacing">
           <number>0</number>
          </property>
          <item>
           <widget class="QLabel" name="uid_label">
            <property name="styleSheet">
             <string notr="true">font-size: 10px</string>
            </property>
            <property name="text">
             <string>UID</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="uid_input"/>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="username_container">
       <property name="title">
        <string/>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_23">
        <property name="leftMargin">
         <number>4</number>
        </property>
        <property name="topMargin">
         <number>8</number>
        </property>
        <property name="rightMargin">
         <number>8</number>
        </property>
        <property name="bottomMargin">
         <number>4</number>
        </property>
        <item>
         <layout class="QVBoxLayout" name="username_container_layout">
          <property name="spacing">
           <number>0</number>
          </property>
          <item>
           <widget class="QLabel" name="username_label">
            <property name="styleSheet">
             <string notr="true">font-size: 10px</string>
            </property>
            <property name="text">
             <string>Username</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="username_input"/>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="status_container">
       <property name="title">
        <string/>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_21">
        <property name="leftMargin">
         <number>4</number>
        </property>
        <property name="topMargin">
         <number>8</number>
        </property>
        <property name="rightMargin">
         <number>8</number>
        </property>
        <property name="bottomMargin">
         <number>4</number>
        </property>
        <item>
         <layout class="QVBoxLayout" name="status_container_layout">
          <property name="spacing">
           <number>0</number>
          </property>
          <item>
           <widget class="QCheckBox" name="status_checkbox">
            <property name="text">
             <string>Is live</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="section_2_layout">
     <item>
      <widget class="QGroupBox" name="password_container">
       <property name="title">
        <string/>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_6">
        <property name="leftMargin">
         <number>4</number>
        </property>
        <property name="topMargin">
         <number>8</number>
        </property>
        <property name="rightMargin">
         <number>8</number>
        </property>
        <property name="bottomMargin">
         <number>4</number>
        </property>
        <item>
         <layout class="QVBoxLayout" name="password_container_layout">
          <property name="spacing">
           <number>0</number>
          </property>
          <item>
           <widget class="QLabel" name="password_label">
            <property name="styleSheet">
             <string notr="true">font-size: 10px</string>
            </property>
            <property name="text">
             <string>Passowrd</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="password_input"/>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="two_fa_container">
       <property name="title">
        <string/>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_9">
        <property name="leftMargin">
         <number>4</number>
        </property>
        <property name="topMargin">
         <number>8</number>
        </property>
        <property name="rightMargin">
         <number>8</number>
        </property>
        <property name="bottomMargin">
         <number>4</number>
        </property>
        <item>
         <layout class="QVBoxLayout" name="two_fa_container_layout">
          <property name="spacing">
           <number>0</number>
          </property>
          <item>
           <widget class="QLabel" name="two_fa_label">
            <property name="styleSheet">
             <string notr="true">font-size: 10px</string>
            </property>
            <property name="text">
             <string>2FA</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="two_fa_input"/>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="section_3_layout">
     <item>
      <widget class="QGroupBox" name="phone_number_container">
       <property name="title">
        <string/>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_14">
        <property name="leftMargin">
         <number>4</number>
        </property>
        <property name="topMargin">
         <number>8</number>
        </property>
        <property name="rightMargin">
         <number>8</number>
        </property>
        <property name="bottomMargin">
         <number>4</number>
        </property>
        <item>
         <layout class="QVBoxLayout" name="phone_number_container_layout">
          <property name="spacing">
           <number>0</number>
          </property>
          <item>
           <widget class="QLabel" name="phone_number_label">
            <property name="styleSheet">
             <string notr="true">font-size: 10px</string>
            </property>
            <property name="text">
             <string>Phone number</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="phone_number_input"/>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="email_container">
       <property name="title">
        <string/>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <property name="leftMargin">
         <number>4</number>
        </property>
        <property name="topMargin">
         <number>8</number>
        </property>
        <property name="rightMargin">
         <number>8</number>
        </property>
        <property name="bottomMargin">
         <number>4</number>
        </property>
        <item>
         <layout class="QVBoxLayout" name="email_container_layout">
          <property name="spacing">
           <number>0</number>
          </property>
          <item>
           <widget class="QLabel" name="email_label">
            <property name="styleSheet">
             <string notr="true">font-size: 10px</string>
            </property>
            <property name="text">
             <string>email</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="email_input"/>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="email_password_container">
       <property name="title">
        <string/>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_17">
        <property name="leftMargin">
         <number>4</number>
        </property>
        <property name="topMargin">
         <number>8</number>
        </property>
        <property name="rightMargin">
         <number>8</number>
        </property>
        <property name="bottomMargin">
         <number>4</number>
        </property>
        <item>
         <layout class="QVBoxLayout" name="email_password_container_layout">
          <property name="spacing">
           <number>0</number>
          </property>
          <item>
           <widget class="QLabel" name="email_password_label">
            <property name="styleSheet">
             <string notr="true">font-size: 10px</string>
            </property>
            <property name="text">
             <string>Email password</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="email_password_input"/>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="section_4_layout">
     <item>
      <widget class="QGroupBox" name="note_container">
       <property name="title">
        <string/>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_30">
        <property name="leftMargin">
         <number>4</number>
        </property>
        <property name="topMargin">
         <number>8</number>
        </property>
        <property name="rightMargin">
         <number>8</number>
        </property>
        <property name="bottomMargin">
         <number>4</number>
        </property>
        <item>
         <layout class="QVBoxLayout" name="note_container_layout">
          <property name="spacing">
           <number>0</number>
          </property>
          <item>
           <widget class="QLabel" name="note_label">
            <property name="styleSheet">
             <string notr="true">font-size: 10px</string>
            </property>
            <property name="text">
             <string>Note</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="note_input"/>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="type_container">
       <property name="title">
        <string/>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_25">
        <property name="leftMargin">
         <number>4</number>
        </property>
        <property name="topMargin">
         <number>8</number>
        </property>
        <property name="rightMargin">
         <number>8</number>
        </property>
        <property name="bottomMargin">
         <number>4</number>
        </property>
        <item>
         <layout class="QVBoxLayout" name="type_container_layout">
          <property name="spacing">
           <number>0</number>
          </property>
          <item>
           <widget class="QLabel" name="type_label">
            <property name="styleSheet">
             <string notr="true">font-size: 10px</string>
            </property>
            <property name="text">
             <string>Type</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="type_input"/>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="group_container">
       <property name="title">
        <string/>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_13">
        <property name="leftMargin">
         <number>4</number>
        </property>
        <property name="topMargin">
         <number>8</number>
        </property>
        <property name="rightMargin">
         <number>8</number>
        </property>
        <property name="bottomMargin">
         <number>4</number>
        </property>
        <item>
         <layout class="QVBoxLayout" name="group_container_layout">
          <property name="spacing">
           <number>0</number>
          </property>
          <item>
           <widget class="QLabel" name="group_label">
            <property name="styleSheet">
             <string notr="true">font-size: 10px</string>
            </property>
            <property name="text">
             <string>Group</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="group_input"/>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="name_container">
       <property name="title">
        <string/>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_33">
        <property name="leftMargin">
         <number>4</number>
        </property>
        <property name="topMargin">
         <number>8</number>
        </property>
        <property name="rightMargin">
         <number>8</number>
        </property>
        <property name="bottomMargin">
         <number>4</number>
        </property>
        <item>
         <layout class="QVBoxLayout" name="name_container_layout">
          <property name="spacing">
           <number>0</number>
          </property>
          <item>
           <widget class="QLabel" name="name_label">
            <property name="styleSheet">
             <string notr="true">font-size: 10px</string>
            </property>
            <property name="text">
             <string>Name</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="name_combobox"/>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Save</set>
     </property>
     <property name="centerButtons">
      <bool>false</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Dialog_Profile</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Dialog_Profile</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
